Frontend/my-vite-app/src/Components/CreateNewAccount.jsx

```diff
@@ -1,35 +1,35 @@
-import { useState } from 'react'
-import { createNewAccount, getMe } from './fetchingData'
-import { useNavigate } from 'react-router-dom'
+import { useState } from 'react';
+import { createNewAccount, getMe } from './fetchingData';
+import { useNavigate } from 'react-router-dom';
 
-const CreateNewAccount = () => {
-  const [name, setName] = useState('')
-  const [surname, setSurname] = useState('')
-  const [email, setEmail] = useState('')
-  const [password, setPassword] = useState('')
-  const [major, setMajor] = useState('')
-  const [classification, setClassification] = useState('')
-  const [profilePicture, setProfilePicture] = useState('')
-  const [imagePreview, setImagePreview] = useState('')
-  const [preferredContact, setPreferredContact] = useState('')
-  const [phone, setPhone] = useState('')
-  const [passwordConfirmation,setPasswordConfirmation] = useState('')
-  const navigate = useNavigate()
+const createNewUserAccount = () => {
```

Comment:
The normal convention for component names is actually capital-case, and I believe the react integration in your eslint config should be automatically enforcing this...